barretenberg/ts/src/barretenberg/index.ts

```diff
@@ -68,8 +68,8 @@ export class Barretenberg extends BarretenbergApi {
 
   async initSRSClientIVC(): Promise<void> {
     // crsPath can be undefined
-    const crs = await Crs.new(2 ** 21 + 1, this.options.crsPath);
-    const grumpkinCrs = await GrumpkinCrs.new(2 ** 16 + 1, this.options.crsPath);
+    const crs = await Crs.new(2 ** 19 + 1, this.options.crsPath);
+    const grumpkinCrs = await GrumpkinCrs.new(2 ** 14 + 1, this.options.crsPath);
```

Comment:
did this go down recently?

Reply:
Idk when these parameters were possible but apparently they are. It's just a question of what we're testing in WASM... eventually these need to be dynamic.